app/react/Uploads/components/specs/__snapshots__/ImportPanel.spec.js.snap

```diff
@@ -1,5 +1,54 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`ImportPanel rendering states error when there is an error should render an error message 1`] = `
```

Comment:
Please update, I think this snapshot is not required now

Reply:
Done

Reply:
Done